index.js

```diff
@@ -1,9 +1,13 @@
 'use strict';
 
-module.exports = function(appId) {
-    return function(request) {
-        request.set('X-Csrf-Token', localStorage['XSRF.Token']);
-        request.set('X-D2L-App-Id', appId);
-        return request;
-    };
+module.exports = function(req) {
+	// This plugin only works for relative URLs. Sending CSRF tokens to foreign
+	// origins would be bad. This plugin is a no-op in those cases.
+	if (req.url[0] != '/') {
+		return req;
```

Comment:
Consider a `console.out` here.  When I misuse `React` in a works-but-not-what-I'm-supposed-to-do kind of way, `React` lets me do it but warns me in the console.